test/e2e/set/scyllacluster/scyllacluster_replace.go

```diff
@@ -72,11 +72,15 @@ var _ = g.Describe("ScyllaCluster", func() {
 		ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
 		defer cancel()
 
-		sc := scyllafixture.BasicScyllaCluster.ReadOrFail()
+		sc := f.GetDefaultScyllaCluster()
 		sc.Spec.Repository = e.scyllaImageRepository
 		sc.Spec.Version = e.scyllaVersion
 		sc.Spec.Datacenter.Racks[0].Members = 3
 
+		if e.procedure == clusterIPProcedure && (sc.Spec.ExposeOptions == nil || sc.Spec.ExposeOptions.NodeService == nil || sc.Spec.ExposeOptions.NodeService.Type == scyllav1.NodeServiceTypeHeadless) {
+			g.Skip("Skipping because ClusterIP-based replace procedure requires member services to have ClusterIPs")
+		}
```

Comment:
nit
```suggestion
		isHeadlessNodeService := sc.Spec.ExposeOptions == nil || sc.Spec.ExposeOptions.NodeService == nil || sc.Spec.ExposeOptions.NodeService.Type == scyllav1.NodeServiceTypeHeadless
		if e.procedure == clusterIPProcedure && isHeadlessNodeService {
			g.Skip("Skipping because ClusterIP-based replace procedure can only be tested on ScyllaClusters exposed on ClusterIPs")
		}
```

Reply:
done

Reply:
the main point was to make the condition smaller, but I don't see the variable extraction done

Reply:
ok, I only noticed the comment at first. done